pkg/autoscaler/autoscaler_server.go

```diff
@@ -129,22 +131,41 @@ func (s *AutoScaler) pollAPIServer() {
 			r := resource.NewQuantity(0, guessFormat(res))
 			r.SetMilli(want)
 			newReqs[ctr].Requests[apiv1.ResourceName(res)] = *r
-			glog.V(0).Infof("Setting %s requests[%q] = %v", ctr, res, r)
+			glog.V(4).Infof("Setting %s requests[%q] = %v", ctr, res, r)
```

Comment:
With below logs being added when updating, may be better to make it clear that this is the calculated requirements (v.s. the being update ones).

Reply:
Updated to "Calculated"